Tests/Editor/TextureTests.cs

```diff
@@ -0,0 +1,58 @@
+using NUnit.Framework;
+using Unity.ProjectAuditor.Editor;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.ProjectAuditor.EditorTests
+{
+    class TextureTests : TestFixtureBase
+    {
+        const int resolution = 1;
+        string currentPlatform = EditorUserBuildSettings.activeBuildTarget.ToString();
+        Texture textureToCompare;
+        TextureImporter textureViaImporter;
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            var texture = new Texture2D(resolution, resolution); //defaults: mipmaps = true & format = automatic
+            texture.SetPixel(0, 0, Random.ColorHSV());
+            texture.name = "ProceduralTextureForTest321.png";
+            texture.Apply();
+
+            var bytes = texture.EncodeToPNG();
+
+            var tempTestTexture = new TempAsset(texture.name, bytes);
+
+            var allTextures = AssetDatabase.FindAssets("t: Texture, a:assets");
+
+            textureToCompare = (Texture)AssetDatabase.LoadAssetAtPath(AssetDatabase.GUIDToAssetPath(allTextures[0]), typeof(Texture));
+            textureViaImporter = (AssetImporter.GetAtPath(AssetDatabase.GUIDToAssetPath(allTextures[0])) as TextureImporter); //Needed since the texture/texture via TextureImporter properties are both used in the TextureModule script to access varying properties.
+        }
+
+        [Test]
+        [Explicit]
+        public void Texture_Properties_AreReported()
+        {
+            var textureTests = Analyze(IssueCategory.Texture);
+
+            Assert.AreEqual(textureToCompare.name, textureTests[0].customProperties[0], "Checked Texture Name");
```

Comment:
you cannot assume your test texture is always `textureTests[0]`. You need to search for it.

Reply:
Now updated to search/id/and use the correct test (procedurally created) texture for comparison.